modules/ROOT/pages/faq/index.adoc

```diff
@@ -1,6 +1,14 @@
 FAQ
 ===
 
+[[I want to upgrade from Community Version to Enterprise Version. What are the changes?]]
```

Comment:
The anchor's quite long, but if you happy with it, then to follow the convention, you need to add hyphens between each word, as in the example below.

Reply:
Do I need this anchor? I just copy pasted from the things that were already there